owncloudApp/src/main/java/com/owncloud/android/authentication/AuthenticatorAsyncTask.java

```diff
@@ -63,6 +60,8 @@ protected RemoteOperationResult doInBackground(Object... params) {
             client.setCredentials(credentials);
 
             // Operation - try credentials
+            String REMOTE_PATH = "/";
+            boolean SUCCESS_IF_ABSENT = false;
```

Comment:
We don't need to use capital letters for this variables now, do we?

Reply:
Yes, you're right